lib/client.js

```diff
@@ -1,29 +1,22 @@
 (function (global, factory, undefined) {
 
-  if ( typeof module === 'object' && typeof module.exports === 'object' ) {
-    var ActivityStreams = require('activity-streams');
+  if ( typeof module === "object" && typeof module.exports === "object" ) {
```

Comment:
I recently read somewhere (can't remember exactly where though), that most people moved away from Crockford style to [Standard JS](https://github.com/feross/standard). And that one recommends single quotes.